### ISSUE TITLE
PDF Preview turns into garbled text or stuck with loading page

### ISSUE THREAD

**Issue by ryukenji3** [tiredofit/docker-libreoffice-online#10]
Hey Tiredofit,

I use this project as the preview server of nextcloud, and everything just works fine (docx/pptx/xlsx/png/) but PDF. PDF files always turns into garbled text(small files) or stuck with loading page(10MB+).

Also tried the official collabora image, basically the same parameters, the official one will be able to preview PDF.

No way out,I turned the log level to **trace** and set it to  **debug mode**, opened only two files with nextcloud, **test.docx & test.pdf**, which are fully documented in this log.

[lool.log](https://github.com/tiredofit/docker-libreoffice-online/files/5764005/lool.log)

my docker-compose.yml like this:

```
version: '3.7'
services:

   libreoffice-app:
      image: tiredofit/libreoffice-online
      container_name: office
      ports:
        - 9980:9980
      cap_add:
        - MKNOD
      environment:
        - ADMIN_USER=admin
        - ADMIN_PASS=admin
        - ALLOWED_HOSTS=*
        - DICTIONARIES="en_GB en_US"
        - LOG_LEVEL=trace
        - LOG_TYPE=FILE
        - DEBUG_MODE=TRUE
        - ENABLE_TLS=FALSE
        - ENABLE_TLS_REVERSE_PROXY=FALSE
        - ENABLE_TLS_CERT_GENERATE=FALSE
        - EXTRA_OPTIONS=--o:ssl.enable=false --o:ssl.termination=false
```

**Comment by ryukenji3** [comment]
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/76910583/103551308-1fa02b80-4ee5-11eb-9039-bb9de81526a7.png">

**Comment by ryukenji3** [comment]
resolved in 2.0.0, thanks @tiredofit 
<img width="1739" alt="image" src="https://user-images.githubusercontent.com/76910583/108616058-fe6bae00-7444-11eb-8459-c8971ac3e056.png">
